manage/organizations/index.rst

```diff
@@ -0,0 +1,146 @@
+Organizations
+=============
+
+Depending on your organization’s IT capabilities,
+organizations can be managed individually or in bulk.
+
+.. figure:: /images/manage/organizations/general-view-of-a-busy-organization.png
+   :alt: Creating and editing users directly in the Admin Panel
+   :align: center
+   :width: 80%
+
+   The simplest way to manage organizations is directly in the Admin Panel.
+
+Learn more about managing organizations...
+
+.. toctree::
+   :maxdepth: 1
+
+   via the Admin Panel <via-the-admin-panel>
+   via CSV import <via-csv-import>
+   via REST API <https://docs.zammad.org/en/latest/api/intro.html>
+
+.. note:: 😲 **Technical Limitations**
+
+   * Organizations currently cannot be removed. The only exception
+     is Zammads :doc:`/system/data-privacy` function.
+   * Unlike users, agents cannot just create new organizations.
+     Check the :doc:`permission reference </manage/roles/admin-permissions>`
+     to learn more.
+   * Because of how organization references work with users, external syncs
+     like LDAP or Exchange *do not* support organization mapping.
+
+        .. hint::
+
+           This is relevant to you? Consider domain based assignments.
+
+.. warning:: 🥵 **BIG organizations can cause performance issues**
+
+   Organizations with many members can cause a fairly high system load in
+   some situations. This especially affects organizations that run many
+   updates or e.g. ticket creations on their members which causes overhead.
```

Comment:
Suggestion to simplify the sentence "This especially affects organizations that run many
   updates or e.g. ticket creations on their members which causes overhead.":

This especially affects organizations that have many active members.

Reply:
I personally think that this does not outline enough what exactly active members means. I'm open for simplifying the sentence, however, administrators have to be aware of performance issues and how they can be caused.

This helps administrators to understand scopes and decide better.

Reply:
Another suggestion:
This especially affects organizations whose members run many updates, for example ticket creations or frequent communication. A lot of linked data syncs may cause an overhead.